theworkoutinfo/ViewModels/HomeViewModel.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  HomeViewModel.swift
+//  theworkoutinfo
+//
+//  Created by Gonzalo Barrios on 4/4/18.
+//  Copyright © 2018 Gonzalo Barrios. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+class HomeViewModel {
+    
+    let disposeBag = DisposeBag()
+    
+    //MARK: - Inputs
+    
+    let tryLogout: AnyObserver<Void>
+    let loadExercises: AnyObserver<Void>
+    
+    //MARK: - Outputs
+    
+    let didLogout: Observable<Void>
+    
+    var exercises: Observable<[Exercise]> {
+        return _exercisesVariable.asObservable()
+    }
+    
+    var error: Observable<String> {
+        return _error.asObservable()
+    }
+    
+    //MARK: Properties
+        
+    private let _error = PublishSubject<String>()
+    private let _exercisesVariable = Variable<[Exercise]>([])
+    
+    init() {
+        let _tryLogout = PublishSubject<Void>()
+        tryLogout = _tryLogout.asObserver()
+        didLogout = _tryLogout.asObservable()
+        
+        let _loadExercises = PublishSubject<Void>()
+        loadExercises = _loadExercises.asObserver()
+        
+        let repositoryExercises = ContentRepository.sharedInstance.getExercisesWithImages()
```

Comment:
I would not define this variable like this. It is confusing since it seems it is an exercises repository, but it is a reference to a method. I would suggest dropping the variable and using the method directly below